.github/scripts/gitlog.sh

```diff
@@ -0,0 +1,10 @@
+#!/bin/bash
+
+LAST_TAG_DESCRIBE=`for tag in $(git tag | grep -E 'v[0-9]+\.[0-9]'); do
+    git describe --tags --long --match="$tag"
+done | sort -k2 -t"-" -n | head -n1`
+
+LAST_TAG=${LAST_TAG_DESCRIBE%%-*}  # retain the part before the first dash
+HEAD=${LAST_TAG_DESCRIBE##*g}
+
+git log $LAST_TAG..HEAD --oneline > docs/unreleased.log
```

Comment:
This makes me wonder if we could generate a full release history or a history of the last N releases rather than clearing this out every time we push a tag. Out of scope for this task, though.

Reply:
Yes, that should be doable if we decide we want to